README.md

```diff
@@ -172,6 +172,19 @@ OPTIONS:
 EXAMPLE: run.sh -s "Squeak-trunk" --headfull /path/to/projects
 ```
 
+## Running locally
+
+If you need to run smalltalkCI locally you can follow these steps:
+
+1. clone your repository
+  * a) from github: `git clone https://github.com/your/project.git`, or
+  * b) clone your local repo: `git clone /wherever/is/your/project`, or
+  * c) use your local repo directly
+2. cd into the cloned directory `cd project`
+3. download smalltalkCI: `wget -O smalltalkCI.zip https://github.com/hpi-swa/smalltalkCI/archive/master.zip`
+4. unzip it: `unzip -o smalltalkCI.zip`
+5. run smalltalkCI, provide dialect you want to test and your local config: `smalltalkCI-master/run.sh --headfull -s "Pharo-5.0" .smalltalk.ston`
+
```

Comment:
Why? So you don't accidentally commit it? Are people not looking at what
they are commiting?

On Thu, Mar 24, 2016 at 12:50 AM, Dale Henrichs <notifications@github.com>
wrote:

> In README.md
> <https://github.com/hpi-swa/smalltalkCI/pull/95#discussion_r57256531>:
>
> > @@ -172,6 +172,19 @@ OPTIONS:
> >  EXAMPLE: run.sh -s "Squeak-trunk" --headfull /path/to/projects
> >  ```
> >
> > +## Running locally
> > +
> > +If you need to run smalltalkCI locally you can follow these steps:
> > +
> > +1. clone your repository
> > +  * a) from github: `git clone https://github.com/your/project.git` <https://github.com/your/project.git>, or
> > +  * b) clone your local repo: `git clone /wherever/is/your/project`, or
> > +  * c) use your local repo directly
> > +2. cd into the cloned directory `cd project`
> > +3. download smalltalkCI: `wget -O smalltalkCI.zip https://github.com/hpi-swa/smalltalkCI/archive/master.zip` <https://github.com/hpi-swa/smalltalkCI/archive/master.zip>
> > +4. unzip it: `unzip -o smalltalkCI.zip`
> > +5. run smalltalkCI, provide dialect you want to test and your local config: `smalltalkCI-master/run.sh --headfull -s "Pharo-5.0" .smalltalk.ston`
> > +
>
> Does it make sense to mention that one should remove the smalltalkCI after
> you're done (if you've downloaded it into your project directory)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hpi-swa/smalltalkCI/pull/95/files/1c8292a4f3dfb585bfee03e5b8768173cd0cf15b#r57256531>
>